plugins/example.ts

```diff
@@ -0,0 +1,30 @@
+import { Plugin, runPlugin } from "./lib/plugin";
+
+type MyPluginParam = {
+  relayerId: string;
+};
+
+async function example(p: Plugin, params: MyPluginParam) {
+  console.log("Plugin started with params:", params);
+
+  /**
+   * Instances the relayer with the given id.
+   */
+  const relayer = p.useRelayer(params.relayerId || "sepolia-example");
+
+  /**
+   * Sends an arbitrary transaction through the relayer.
```

Comment:
Marcos mentioned we will introduce wait method.

Maybe before we introduce it we should denote that sendTransaction returns id and does not wait for tx to be mined.